ansible/roles/erwin-idm/templates/init_job.yml.j2

```
@@ -0,0 +1,32 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: erwinidm-init-job
+  labels:
+    app: api
```

api is the wrong label for this job.
```suggestion
    app: erwinidm
```

---

Done